src/applications/vaos/tests/mocks/setup.js

```diff
@@ -474,3 +481,87 @@ export async function setPreferredDate(store, preferredDate) {
 
   return screen.history.push.firstCall.args[0];
 }
+
+/**
+ * Set up community care flow for use in page tests from
+ * the calendar page forward.
+ *
+ * @export
+ * @param {Object} params
+ * @param {Object} toggles Any feature toggles to set. CC toggle is set by default
+ * @param {Array<Object>} parentSites List of parent sites and data, in the format used
+ *  by the getV2FacilityMock params, so you can pass name, id, and address
+ * @param {?Array<string>} supportedSites List of site ids that support community care.
+ *  Defaults to the parent site ids if not provided
+ * @param {?Array<string>} registeredSites List of registered site ids. Will use ids
+ *  from parentSites list if not provided
+ * @param {string} typeOfCareId Type of care id string to use. Use V2 id format (idV2 in
+ *  type of care list)
+ * @param {Object} [residentialAddress=null] VA Profile address to use for the user
+ * @returns {ReduxStore} Redux store with data set up
+ */
+export async function setCommunityCareFlow({
```

Comment:
Based on Kevin's recommendations from the mock data spike, made a higher level helper here for use in several tests. There are definitely some other places where we could use this, too.

Reply:
👍 Looking at this method I think I might be able to put some of the `setTypeOf` helpers into my spike function.

This setup file also seems like a good place to put a lot of the higher level helpers if they're shared across tests.